tests/test_dialect.py

```diff
@@ -9,3 +9,34 @@ def test_dialect(host, http_path, token):
     )
     tables = inspect(engine).get_table_names()
     print(tables)
+
+
+def test_has_table(host, http_path, token):
+    engine = create_engine(
+        f"databricks+connector://token:{token}@{host}:443/default",
+        connect_args={"http_path": f"{http_path}"},
+    )
+
+    engine.dialect.has_table(engine, 'table_name_that_should_not_exist')
+
+
+def test_create_table_with_primary_key():
+    def raise_if_primary_key(sql, *multiparams, **params):
+        ddl_statement =  str(sql.compile(dialect=engine.dialect))
+        if 'primary' in ddl_statement.lower():
+            raise ValueError(f'{ddl_statement=}')
```

Comment:
Locally I changed this function to something like this to help see the generated DDL statements, and assert on anything necessary:

```python
    def ddl_assertions(sql, *multiparams, **params):
        ddl_statement = str(sql.compile(dialect=engine.dialect))
        print(ddl_statement)
        assert "PRIMARY KEY" not in ddl_statement
```